### ISSUE TITLE
Failed scrape doesn't exit

### ISSUE THREAD

**Issue by rgaudin** [webrecorder/browsertrix-crawler#359]
In a zimit run using `0.10.4` crawler, I have a run that halted with a Timeout then nothing. Process hasn't exited but there's no output anymore.

```
{"logLevel":"error","timestamp":"2023-08-27T19:46:23.915Z","context":"worker","message":"Worker Exception","details":{"type":"exception","message":"Navigation timeout of 90000 ms exceeded","stack":"TimeoutError: Navigation timeout of 90000 ms exceeded\n    at Timeout.<anonymous> (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/Deferred.js:24:33)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)","page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:46:23.916Z","context":"pageStatus","message":"Page Load Failed","details":{"loadState":0,"page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}
```

**Comment by ikreymer** [comment]
@rgaudin were there additional workers running besides just one? Probably there is another work that hasn't failed yet (may be stuck). Is there more to the log?

**Comment by rgaudin** [comment]
Sorry about this very short log, here's the full thing

```
Running browsertrix-crawler crawl: crawl --failOnFailedSeed --waitUntil load --depth -1 --timeout 90 --behaviors autoplay,autofetch,siteSpecific --behaviorTimeout 90 --sizeLimit 4294967296 --diskUtilization 90 --timeLimit 7200 --url https://www.helpforibs.com/ --userAgentSuffix Youzim.it+contact+zimfarm@kiwix.org --cwd /output/.tmpxc8v4i8q --statsFilename /output/crawl.json
{"logLevel":"info","timestamp":"2023-08-27T19:32:17.688Z","context":"general","message":"Browsertrix-Crawler 0.10.4 (with warcio.js 1.6.2 pywb 2.7.4)","details":{}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:17.692Z","context":"general","message":"Seeds","details":[{"url":"https://www.helpforibs.com/","include":["/^https?:\\/\\/www\\.helpforibs\\.com\\//"],"exclude":[],"scopeType":"prefix","sitemap":false,"allowHash":false,"maxExtraHops":0,"maxDepth":1000000}]}
{"logLevel":"info","timestamp":"2023-08-27T19:32:19.865Z","context":"worker","message":"Creating 1 workers","details":{}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:19.867Z","context":"worker","message":"Worker starting","details":{"workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:20.397Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/"}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:20.402Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":null,"total":null,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:32:19.873Z\",\"url\":\"https://www.helpforibs.com/\",\"added\":\"2023-08-27T19:32:18.257Z\",\"depth\":0}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:21.387Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:38.862Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"tel:18666404942","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:38.881Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetForm'))","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:38.916Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetDTForm'))","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:38.924Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('NewsletterForm'))","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:38.930Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetFormPop'))","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:38.959Z","context":"behavior","message":"Running behaviors","details":{"frames":1,"frameUrls":["https://www.helpforibs.com/"],"page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:38.961Z","context":"behavior","message":"Run Script Started","details":{"frameUrl":"https://www.helpforibs.com/","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:38.967Z","context":"behavior","message":"Run Script Finished","details":{"frameUrl":"https://www.helpforibs.com/","page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:38.968Z","context":"behavior","message":"Behaviors finished","details":{"finished":1,"page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:38.970Z","context":"pageStatus","message":"Page Finished","details":{"loadState":4,"page":"https://www.helpforibs.com/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:39.007Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/footer/"}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:39.009Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":1,"total":78,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:32:39.006Z\",\"url\":\"https://www.helpforibs.com/footer/\",\"added\":\"2023-08-27T19:32:38.861Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:39.895Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:48.848Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"tel:18666404942","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:48.863Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetForm'))","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:48.898Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetDTForm'))","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:32:48.904Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('NewsletterForm'))","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.944Z","context":"behavior","message":"Running behaviors","details":{"frames":1,"frameUrls":["https://www.helpforibs.com/footer/"],"page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.945Z","context":"behavior","message":"Run Script Started","details":{"frameUrl":"https://www.helpforibs.com/footer/","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.951Z","context":"behavior","message":"Run Script Finished","details":{"frameUrl":"https://www.helpforibs.com/footer/","page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.951Z","context":"behavior","message":"Behaviors finished","details":{"finished":1,"page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.952Z","context":"pageStatus","message":"Page Finished","details":{"loadState":4,"page":"https://www.helpforibs.com/footer/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.985Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/"}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:48.989Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":2,"total":85,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:32:48.985Z\",\"url\":\"https://www.helpforibs.com/diet/\",\"added\":\"2023-08-27T19:32:38.863Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:32:49.776Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.558Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"tel:18666404942","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.606Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetForm'))","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.706Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetDTForm'))","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.733Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('NewsletterForm'))","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.746Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('DietForm'))","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:33:03.758Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetSmallForm'))","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:03.942Z","context":"behavior","message":"Running behaviors","details":{"frames":1,"frameUrls":["https://www.helpforibs.com/diet/"],"page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:03.943Z","context":"behavior","message":"Run Script Started","details":{"frameUrl":"https://www.helpforibs.com/diet/","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:03.957Z","context":"behavior","message":"Run Script Finished","details":{"frameUrl":"https://www.helpforibs.com/diet/","page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:03.957Z","context":"behavior","message":"Behaviors finished","details":{"finished":1,"page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:03.959Z","context":"pageStatus","message":"Page Finished","details":{"loadState":4,"page":"https://www.helpforibs.com/diet/","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:04.033Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/trigger1.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:04.036Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":3,"total":85,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:33:04.031Z\",\"url\":\"https://www.helpforibs.com/diet/trigger1.asp\",\"added\":\"2023-08-27T19:32:38.865Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:33:04.869Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:34.870Z","context":"general","message":"Page Loading Slowly, skipping behaviors","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:37.403Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"tel:18666404942","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:37.419Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetForm'))","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:37.442Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetDTForm'))","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:37.446Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('NewsletterForm'))","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:34:37.463Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetSmallForm'))","page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:34:37.486Z","context":"behavior","message":"Skipping behaviors for slow page","details":{"page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:34:37.487Z","context":"pageStatus","message":"Page Finished","details":{"loadState":3,"page":"https://www.helpforibs.com/diet/trigger1.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:34:37.519Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/trigger2.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:34:37.522Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":4,"total":88,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:34:37.518Z\",\"url\":\"https://www.helpforibs.com/diet/trigger2.asp\",\"added\":\"2023-08-27T19:32:38.866Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:34:38.161Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:36:08.208Z","context":"general","message":"Page Loading Slowly, skipping behaviors","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:36:10.739Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"tel:18666404942","page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:36:10.756Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetForm'))","page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:36:10.782Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('CheatSheetDTForm'))","page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:36:10.788Z","context":"general","message":"Invalid Page - URL must start with http:// or https://","details":{"url":"javascript:void(UPTsubmitform('NewsletterForm'))","page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:36:10.824Z","context":"behavior","message":"Skipping behaviors for slow page","details":{"page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:36:10.825Z","context":"pageStatus","message":"Page Finished","details":{"loadState":3,"page":"https://www.helpforibs.com/diet/trigger2.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:36:11.033Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/fiber1.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:36:11.035Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":5,"total":88,"pending":1,"failed":0,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:36:10.854Z\",\"url\":\"https://www.helpforibs.com/diet/fiber1.asp\",\"added\":\"2023-08-27T19:32:38.868Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:36:11.694Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/fiber1.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:37:41.715Z","context":"general","message":"Page Load Timeout, skipping page","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/fiber1.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:37:41.715Z","context":"worker","message":"Worker Exception","details":{"type":"exception","message":"Navigation timeout of 90000 ms exceeded","stack":"TimeoutError: Navigation timeout of 90000 ms exceeded\n    at Timeout.<anonymous> (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/Deferred.js:24:33)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)","page":"https://www.helpforibs.com/diet/fiber1.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:37:41.716Z","context":"pageStatus","message":"Page Load Failed","details":{"loadState":0,"page":"https://www.helpforibs.com/diet/fiber1.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:37:41.745Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/fiber2.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:37:41.748Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":6,"total":88,"pending":1,"failed":1,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:37:41.744Z\",\"url\":\"https://www.helpforibs.com/diet/fiber2.asp\",\"added\":\"2023-08-27T19:32:38.869Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:37:42.401Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/fiber2.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:39:12.426Z","context":"general","message":"Page Loading Slowly, skipping behaviors","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/fiber2.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:39:17.432Z","context":"general","message":"Promise timed out","details":{"seconds":5}}
{"logLevel":"error","timestamp":"2023-08-27T19:39:24.938Z","context":"general","message":"Link extraction timed out","details":{"seconds":5,"page":"https://www.helpforibs.com/diet/fiber2.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:41:51.780Z","context":"worker","message":"Page Worker Timeout","details":{"seconds":250,"workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:41:51.781Z","context":"pageStatus","message":"Page Finished","details":{"loadState":2,"page":"https://www.helpforibs.com/diet/fiber2.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:41:51.807Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/how_to_eat.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:41:51.809Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":7,"total":88,"pending":1,"failed":1,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:41:51.806Z\",\"url\":\"https://www.helpforibs.com/diet/how_to_eat.asp\",\"added\":\"2023-08-27T19:32:38.870Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:41:52.463Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/how_to_eat.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:43:22.468Z","context":"general","message":"Page Load Timeout, skipping page","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/how_to_eat.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:43:22.468Z","context":"worker","message":"Worker Exception","details":{"type":"exception","message":"Navigation timeout of 90000 ms exceeded","stack":"TimeoutError: Navigation timeout of 90000 ms exceeded\n    at Timeout.<anonymous> (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/Deferred.js:24:33)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)","page":"https://www.helpforibs.com/diet/how_to_eat.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:43:22.469Z","context":"pageStatus","message":"Page Load Failed","details":{"loadState":0,"page":"https://www.helpforibs.com/diet/how_to_eat.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:43:22.495Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/10_commandments.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:43:22.497Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":8,"total":88,"pending":1,"failed":2,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:43:22.494Z\",\"url\":\"https://www.helpforibs.com/diet/10_commandments.asp\",\"added\":\"2023-08-27T19:32:38.872Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:43:23.191Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/10_commandments.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:44:53.226Z","context":"general","message":"Page Load Timeout, skipping page","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/10_commandments.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:44:53.227Z","context":"worker","message":"Worker Exception","details":{"type":"exception","message":"Navigation timeout of 90000 ms exceeded","stack":"TimeoutError: Navigation timeout of 90000 ms exceeded\n    at Timeout.<anonymous> (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/Deferred.js:24:33)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)","page":"https://www.helpforibs.com/diet/10_commandments.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:44:53.228Z","context":"pageStatus","message":"Page Load Failed","details":{"loadState":0,"page":"https://www.helpforibs.com/diet/10_commandments.asp","workerid":0}}
{"logLevel":"info","timestamp":"2023-08-27T19:44:53.256Z","context":"worker","message":"Starting page","details":{"workerid":0,"page":"https://www.helpforibs.com/diet/what_to_eat.asp"}}
{"logLevel":"info","timestamp":"2023-08-27T19:44:53.260Z","context":"crawlStatus","message":"Crawl statistics","details":{"crawled":9,"total":88,"pending":1,"failed":3,"limit":{"max":0,"hit":false},"pendingPages":["{\"seedId\":0,\"started\":\"2023-08-27T19:44:53.255Z\",\"url\":\"https://www.helpforibs.com/diet/what_to_eat.asp\",\"added\":\"2023-08-27T19:32:38.873Z\",\"depth\":1}"]}}
{"logLevel":"info","timestamp":"2023-08-27T19:44:53.911Z","context":"general","message":"Awaiting page load","details":{"page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:46:23.914Z","context":"general","message":"Page Load Timeout, skipping page","details":{"msg":"Navigation timeout of 90000 ms exceeded","page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}
{"logLevel":"error","timestamp":"2023-08-27T19:46:23.915Z","context":"worker","message":"Worker Exception","details":{"type":"exception","message":"Navigation timeout of 90000 ms exceeded","stack":"TimeoutError: Navigation timeout of 90000 ms exceeded\n    at Timeout.<anonymous> (file:///app/node_modules/puppeteer-core/lib/esm/puppeteer/util/Deferred.js:24:33)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)","page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}
{"logLevel":"warn","timestamp":"2023-08-27T19:46:23.916Z","context":"pageStatus","message":"Page Load Failed","details":{"loadState":0,"page":"https://www.helpforibs.com/diet/what_to_eat.asp","workerid":0}}

```

As you can see, last update dates 2023-08-27@19:46. I had to kill the process the next day 2023-08-28@15:39

**Comment by ikreymer** [comment]
Similar to other issues, I believe this should be fixed in 0.11.1, please reopen if happens again with latest log.
